checkstyle/import-control.xml

```diff
@@ -9,6 +9,7 @@
     <allow pkg="java"/>
     <allow pkg="javax.management"/>
     <allow pkg="javax.annotation"/>
+    <allow pkg="com.sun"/>
```

Comment:
I am a bit concerned with allowing this globally. You only use it in one project/package. Can we modify this file to restrict the `com.sun` to only the package where its used? The `pkg="io.pravega"` suggests this is possible.